content/definition.it.md

```diff
@@ -1,14 +1,44 @@
 +++
-title = "La definizione di Sorgente Etica"
+title = "La definizione di Sorgente Etica (SE)"
 template = "definition"
+heading="La definizione di Sorgente Etica"
 +++
 
-Un Software è considerato Sorgente Etica (_Ethical Source_) se rispetta i seguenti criteri:
+Un Software è considerato **Open-Source Etico** (EOS) se rispetta *tutti* i seguenti criteri:
 
-1. E' liberamente distribuito con il codice sorgente e puó essere utilizzato o combinato con altri software senza alcuna licenza o prezzo.
-2. E' sviluppato pubblicamente e accetta contributi dalla comunitá.
-3. La sua community è governata da un codice di condotta che è consistente ed imposta.
-4. I suoi creatori hanno il diritto di proibire il suo utilizzo a individui oppure organizzazioni che violano i diritti umani o hanno comporamenti considerati non etici.
-5. I suoi creatori hanno il diritto di sollecitare compensazioni ragionevoli e volontarie dalla comunitá o istituzioni che beneficiano dall'utilizzo del software.
+1. **È di comune beneficio.** La licenza con la quale il software è distribuito non ne proibisce la modifica, la derivazione e il linkaggio o la compilazione con altro software (se non in virtù di una disposizione copyleft), nello spirito generale della distribuzione open source.
+2. **È creato apertamente** Il codice sorgente deve essere disponibile pubblicamente, sviluppato e mantenuto in una visione pubblica ed accoglie la contribuzione pubblica (soggetta alla revisione ed all'approvazione di uno dei mantenitori del progetto.)
+3. **La sua comunitá è accogliente e giusta.** La comunitá  di sviluppatori e contributori del progetto software deve essere governata da un codice di condotta comprensibile e che viene consistentemente ed applicato equamente.
+4. **Pone l'accessibilitá al primo posto.** Se il software è dotato di una interfaccia, deve essere disegnato con l'idea di essere accessibile, assicurando che la funzionalitá del software sia disponibile a tutti gli utenti, inclusi quelli che utilizzano dispositivi assistivi.
```

Comment:
`dispositivi assistivi` means directly that, because there's also the wording `Tecnologia Assistiva` and all the first results on search engines points to devices for people with disabilities